### DISCUSSION TITLE
How does the "Modchart path" thing work

### DISCUSSION THREAD

**Discussion by chaostheunderscore** [Leather128/LeathersFunkinEngine#52] [Q&A] [answered]
like if you put the modchart in with the difficulties what do you write int o the modchart path?

**Comment by Leather128** [comment, marked as answer]
no, what you do is you put the modchart lua file in `data/modcharts/` and then the modchart path is the name (or sometimes path if it's in a folder) (without .lua) to the modchart.

**Reply by chaostheunderscore** [reply]
> no, what you do is you put the modchart lua file in `data/modcharts/` and then the modchart path is the name (or sometimes path if it's in a folder) (without .lua) to the modchart.

so basically, you put the modchart in the modchart and you name it modchartsong and you put modchartsong in the modchart path, correct?

**Reply by Leather128** [reply]
Mod charts go in 

![image](https://user-images.githubusercontent.com/81660872/142772713-5c187475-9314-439d-99f8-7ac0c0c0bb54.png)

And you would type

![image](https://user-images.githubusercontent.com/81660872/142772733-58976cb1-f8ff-45bf-8ecb-8eba039c5d19.png)

In the modchart path (without the quotation marks)
